src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/resource/AssessmentResourceController.kt

```diff
@@ -66,7 +70,13 @@ class AssessmentResourceController(private val assessmentService: AssessmentServ
     @PathVariable("nomsId")
     @Parameter(required = true)
     nomsId: String,
-  ) = assessmentService.getAssessmentByNomsId(nomsId)
+    @Schema(hidden = true)
+    @RequestHeader("Authorization")
+    auth: String,
+  ): AssessmentEntity? {
+    auditService.audit(AuditAction.GET_ASSESSMENT, nomsId, auth, null)
```

Comment:
We only need to audit editing data e.g. POST/PUT/PATCH/DELETE requests. We don't need this for any GETs for now.

Reply:
DONE!